src/target/stm32mp15.c

```diff
@@ -62,16 +65,22 @@
 /* Taken from CM4ROM_PIDRx in 2.3.21 of ES0438 rev 7, pg18 */
 #define ID_STM32MP15x_ERRATA 0x450U
 
+#define SWO_BASE 0xe0083000
+#define SWO_ACPR (SWO_BASE + 0x00010)
+#define SWO_SPPR (SWO_BASE + 0x000f0)
+
 typedef struct stm32mp15_priv {
 	uint32_t dbgmcu_ctrl;
 } stm32mp15_priv_s;
 
 static bool stm32mp15_uid(target_s *target, int argc, const char **argv);
 static bool stm32mp15_cmd_rev(target_s *target, int argc, const char **argv);
+static bool stm32mp15_cmd_swo(target_s *target, int argc, const char **argv);
 
 const command_s stm32mp15_cmd_list[] = {
 	{"uid", stm32mp15_uid, "Print unique device ID"},
 	{"revision", stm32mp15_cmd_rev, "Returns the Device ID and Revision"},
+	{"mp15_swo", stm32mp15_cmd_swo, "Set up SWO mode <1/2> and divisor <0x42>"},
```

Comment:
Think it's fine to name this command just `swo` instead of `mp15_swo`, or `conf_swo` if you still want to prefix it.

Reply:
Correction to this - now that `mon swo` is taken by the in-probe SWO implementation, either `mp15_swo` or `conf_swo` - think the latter is a little more clear as to what it does, but both are acceptable.

Reply:
Renamed to `conf_swo` in hopes that h7/h5 get a similar command. OTOH it is redundant if C5ACCE55 is used (possibly on PE-visible alias at 0x50083000). This is for orbuculum gdbtrace.init scripts, not BMF executable code (historically, at least)